docs/features/software-catalog/well-known-annotations.md

```diff
@@ -0,0 +1,116 @@
+---
+id: well-known-annotations
+title: Well-known Annotations on Catalog Entities
+sidebar_label: Well-known Annotations
+---
+
+This section lists a number of well known
```

Comment:
This is a great start but I would like to see some more zoomed out and conceptually information. It should answer questions like:

* What are the purpose(s) of these annotations? 
* Who is defining them? 
* Why have well-known ones?
* How do I reason about namespacing?
* How are they consumed by plugins?
* Is this the only way to pass information between an entity and the plugin?

Maybe they even deserve it's own doc: `associating-plugins-to-exteranal-sources` or something similar.

Reply:
Yes, I intended that to be a separate page. This one has only a very specific purpose, and it links out to just one other page (the descriptor format) for now, but more should be added.